src/nhex_bass_hit.cpp

```diff
@@ -144,8 +144,8 @@ void NHexBassHit::update(void)
       }
       else
       {
+        env_lin_current -= env_decrement;
```

Comment:
this is very important to not do everytime! this fix removes even more of the click